bedrock/firefox/templates/firefox/browsers/chromebook.html

```diff
@@ -0,0 +1,92 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import feature_card, hero with context %}
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('browsers-chromebook-get-firefox-browser') }}{% endblock %}
+
+{% block page_desc %} {{ ftl('browsers-chromebook-so-youve-got') }} {% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('chromebook') }}
+{% endblock %}
+
+{% block content %}
+<div class="mzp-l-content">
+  {% call hero(
+    title=ftl('browsers-chromebook-get-firefox-browser'),
+    class='mzp-has-image mzp-t-firefox mzp-t-product-firefox',
+    image_url='img/firefox/browsers/quantum/browser.jpg',
+    heading_level=1
+  ) %}
+  <p>{{ ftl('browsers-chromebook-so-youve-got') }}</p>
+
+  <div class="mzp-c-menu-list mzp-t-cta mzp-t-download" id="chromebook-download-list">
+    <h4 class="mzp-c-menu-list-title">{{ ftl('browsers-chromebook-dropdown-copy') }}</h4>
+    <ul class="mzp-c-menu-list-list download-platform-list">
+      <li class="mzp-c-menu-list-item">
+        <a href="{{ firefox_adjust_url('android', 'firefox-browsers-faq') }}" rel="external noopener" data-link-type="download" data-display-name="Android" data-download-version="android" data-download-os="Android">
+          {{ ftl('browsers-chromebook-get-firefox-for') }}
+          <p><small>{{ ftl('browsers-chromebook-x86-based-chromebook') }}</small></p>
+        </a>
+      </li>
+      <li class="mzp-c-menu-list-item t-sumo">
+        <a href="https://support.mozilla.org/kb/run-firefox-chromeos" data-display-name="Get Firefox Desktop for Chromebook">
```

Comment:
Not sure about the lone `data-display-name` property here without any others. I think we normally use this on download links?